filesystem/Upload.php

```diff
@@ -306,6 +306,15 @@ public function getErrors() {
 class Upload_Validator {
 
 	/**
+	* Contains a list of the max file sizes shared by
+	* all upload fields. This is then duplicated into the
+	* "allowedMaxFileSize" instance property on construct.
+	*
+	* @var array
+	*/
+	private static $default_max_file_size = array();
```

Comment:
I think this variable naming is confusing. It's actually default max file sizes (by extension)